recipes/SHARPlib/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "SHARPlib" %}
+{% set version = "1.0.0rc4" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
@ocefpaf I took a look at the document linked, are there any specific steps I need to take before this can be merged? As far as I gathered reading it, rc status isn't forbidden or anything, but this is my first time packaging for conda. I'm the maintainer of the library being packaged, and it's rc status is primarily being held up by me making sure it can be built and packaged for PyPI and conda-forge. It should not remain in rc status too much longer past that goalpost.

Reply:
Please see https://github.com/conda-forge/cfep/blob/main/cfep-05.md#specification

Reply:
@ocefpaf Thank you, and pardon my lack of clarity. I have been following the[ documentation on contributing here](https://conda-forge.org/docs/maintainer/adding_pkgs/), but can't find anything on ```conda_build_config.yaml```. I was using grayskull to auto-generate my ```meta.yaml```.

Is it sufficient to add a ```conda_build_config.yaml``` with the following contents? Does it need more, or is this + meta.yaml sufficient?
```yaml
channel_targets:
    - conda-forge sharplib_rc
```

Thank you for your patience and assistance

Reply:
Looks like I need to append _rc to the jinja name as well

Reply:
@ocefpaf assuming I understood you correctly, this has been addressed and pushed :smile:

Reply:
> Is it sufficient to add a conda_build_config.yaml with the following contents? Does it need more, or is this + meta.yaml sufficient?

Yep. When updstream issue a stable release, please drop that file and re-render to publish this in the main channel.